<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>121</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QToolButton" name="tbtn2">
      <property name="text">
       <string>桌2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd4">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd5">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd6">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>121</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QToolButton" name="tbtn3">
      <property name="text">
       <string>桌3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd7">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd8">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd9">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>121</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QToolButton" name="tbtn4">
      <property name="text">
       <string>桌4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd10">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd11">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd12">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label11">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label12">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>0</y>
     <width>121</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QToolButton" name="tbtn5">
      <property name="text">
       <string>桌5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd13">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd14">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd15">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label13">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label14">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label15">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>0</y>
     <width>121</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QToolButton" name="tbtn6">
      <property name="text">
       <string>桌6</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd16">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd17">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd18">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label16">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label17">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label18">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>121</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QToolButton" name="tbtn1">
      <property name="text">
       <string>桌1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd1">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd2">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd3">
      <property name="digitCount">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
